### DISCUSSION TITLE
Multiple TraceIDs in Trace Panel

### DISCUSSION THREAD

**Discussion by passarela** [grafana/grafana#66995] [Dashboards, Panels & Visualizations]
hi

Today the "Traces" panel only works if the Query type "TraceID" or "JSON File".

My idea would be to work with Query Type "Search" where it would return all TracesID in the specified period. Similar to the visualization that I have directly through the Jaeger Interface, but in a grafana Panel.
Today the "Traces" panel only works if the Query type "TraceID" or "JSON File".

![chrome_8tR4Akv3VJ](https://user-images.githubusercontent.com/72925771/233449135-5cfbf81e-9b53-4814-a43f-3db78b05039f.png)

![image](https://user-images.githubusercontent.com/72925771/233449407-e91140cd-b758-409b-bdbf-220140ec2305.png)

I intend to provide tracing of certain services for groups of users with access to the panel, releasing the "Explorer" for all views is not a solution, as datasources do not have query permissions.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This feature has been [implemented](https://play.grafana.org/goto/03ttQQhSR?orgId=1). If you are looking for something else, feel free to make an [issue](https://github.com/grafana/grafana/issues) if you would like to request the feature.
